### PR TITLE
Add warning for bad meta description length (SEO)

### PR DESCRIPTION
**Pull request by danielfdickinson**
**What does this PR change? What problem does it solve?**

Warn on meta description <25 characters or >160 characters as this is
the range search engines prefer (according to Bing Webmaster Tools).

As I have added new strings I have made them translatable and provided the ``en`` translation. I haven't been able to figure out how to create a PR on the Wiki in order to provide an update to the list of translated strings there.

**Was the change discussed in an issue or in the Discussions before?**

No, but #521 discusses a draft PR #497 which was determined to not be appropriate as it missed translation glue for new strings and automated changing the description, which was deemed inappropriate as the blogger / site author should write a good description themselves.

It is still useful to know if one has this issue (especially when first switching to PaperMod) at build time, therefore I have added this warning in the event the (SEO) issue exists.

Finally, I wonder if you would like me to update the formatting of the description block, since it's no longer embedded inside the meta content attribute?

## PR Checklist

- [X] This change adds/updates translations and I have used the [template present here](https://github.com/adityatelange/hugo-PaperMod/wiki/Translations#want-to-add-your-language-).
- [X] I have enabled [maintainer edits for this PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [X] This change **does not** include any CDN resources/links.
- [X] This change **does not** include any unrelated scripts such as bash and python scripts.

### CODE REVIEW

**Comment by adityatelange** [comment]
Why would you like to provide warnings only for "description"?
There are a lot more things a user could miss to write to get "bad SEO"
For ex. Date, tags, or maybe forget to add a title?

**tl;dr**: Description is not everything a search engine needs to rank

**Comment by danielfdickinson** [comment]
Well, I was planning to add more checks, but I didn't want to add unrelated changes. In addition, at least for myself, the ones that SEO checkers from Bing and Google complain about are description and title length (of things that I'd want to check for).
I'm happy to keep this in my own tree as a convenience function so that I have a straightforward way to make sure the description (and will be adding title) are within the recommended lengths, because I find it easy to go over the length (more so than under).

**Comment by adityatelange** [comment]
> Well, I was planning to add more checks, but I didn't want to add unrelated changes. In addition, at least for myself, the ones that SEO checkers from Bing and Google complain about are description and title length (of things that I'd want to check for).
> I'm happy to keep this in my own tree as a convenience function so that I have a straightforward way to make sure the description (and will be adding title) are within the recommended lengths, because I find it easy to go over the length (more so than under).

The thing is we cannot make users tell each and everything that they are missing or doing it the wrong way. If you don't put an appropriate title WordPress is not gonna complain about it, rather it's the user's choice.
Hugo SSG is intended to be used by intermediate users and not non-technical people. Hope you understand.

**Comment by danielfdickinson** [comment]
Not a problem. It's more a convenience thing for me that I thought might be useful for others.
